tests/rules/rules-list.directive.spec.js

```diff
@@ -0,0 +1,276 @@
+describe('app.states.RulesListDirective', function() {
+  var $scope;
+  var $compile;
+  var element;
+  var successResponse = {
+    message: 'Success!'
+  };
+
+  beforeEach(function () {
+    module('app.services', 'app.config', 'app.states', 'gettext');
+    bard.inject('RulesState', '$state', 'Session', '$httpBackend');
+  });
+
+  beforeEach(inject(function (_$compile_, _$rootScope_, _$document_) {
+    $compile = _$compile_;
+    $scope = _$rootScope_;
+    $document = _$document_
+  }));
+
+  var compileHTML = function (markup, scope) {
+    element = angular.element(markup);
+    $compile(element)(scope);
+
+    scope.$digest();
+  };
+
+  beforeEach(function () {
+    Session.create({
+      auth_token: 'b10ee568ac7b5d4efbc09a6b62cb99b8',
```

Comment:
I have not dug into this, so take this at face value, but do we want a token hard coded (again, not sure if this is doesn't matter here)